### DISCUSSION TITLE
GoPro 9-x WIFI Command delete/remove pictures and movies missing

### DISCUSSION THREAD

**Discussion by afriess** [KonradIT/goprowifihack#285] [Q&A] [answered]
The title "GoPro 9-x WIFI Command delete/remove pictures and movies missing" is the question. I see the Quick App can remove pictures and movies from the cam, but i found not WIFI media Command to do it. In the documentation here and in Open GoPro i found  a lot of stuff, but nothing about the deletion. 

Does somebody know the command ?

**Comment by KonradIT** [comment, marked as answer]
https://github.com/KonradIT/goprowifihack/blob/9be2d2f006258cfd5b8138cd106539f5b95fe3f6/HERO7/HERO7-Commands.md#media-commands
